### PR TITLE
Feature/location config

### LINKED ISSUE #6: Implement locationconfig trigger

### PR DESCRIPTION
**Pull request by Lauraducky**
Fixes #6

### CODE REVIEW

File: src/PockyBot.NET/Services/Triggers/LocationConfig.cs [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        {
+            var fullText = message.MessageParts.Skip(1).Select(x => x.Text);
+            var commands = string.Join("", fullText)
+                .Split(new[] { ' ' }, StringSplitOptions.RemoveEmptyEntries);
+
+            if (commands.Length < 2)
+            {
+                return new Message
+                {
+                    Text = "Please specify a command. Possible values are get, set and delete."
```
Comment (jimcullenaus) [inline src/PockyBot.NET/Services/Triggers/LocationConfig.cs]: Oxford comma pls

File: src/PockyBot.NET/Services/Triggers/LocationConfig.cs [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                    response = GetLocationMessage(locations);
+                    break;
+                case Actions.Set:
+                    response = await SetLocation(message, commands, locations);
+                    break;
+                case Actions.Delete:
+                    response = await DeleteLocation(message, commands, locations);
+                    break;
+                default:
+                    response = "Unknown command. Possible values are get, set and delete.";
```
Comment (jimcullenaus) [inline src/PockyBot.NET/Services/Triggers/LocationConfig.cs]: ^

**Comment by jimcullenaus** [comment]
Also, what's the value in having this if user locations haven't been implemented alongside it?

**Comment by Lauraducky** [comment]
> Also, what's the value in having this if user locations haven't been implemented alongside it?

We're turning this into a git flow situation so this PR is going into a release branch until userlocation is ready

File: README.md [hunk 3]
```diff
@@ -188,6 +188,15 @@ commands.
...
+- `@PockyBot rotation` — get the snack buying rotation.
+- `@PockyBot welcome` — welcome users.
+- `@PockyBot locationconfig get|set|delete <location>` — manage locations.
+
+#### Admin-only commands
+
+These commands require special permissions to use.
+
 - `@PockyBot finish` — get the results of the cycle.
 - `@PockyBot reset` — clear all pegs from the last cycle.
```
Comment (AmreenC) [inline README.md]: add locationconfig in list for admin-only command

File: README.md [hunk 4]
```diff
@@ -188,6 +188,15 @@ commands.
 - `@PockyBot ping` — verify that the bot is alive.
 - `@PockyBot peg @Person <reason>` — give someone a peg.
 - `@PockyBot status` — get the list of pegs you have given.
+- `@PockyBot help` — get help with using the bot.
+- `@PockyBot rotation` — get the snack buying rotation.
+- `@PockyBot welcome` — welcome users.
+- `@PockyBot locationconfig get|set|delete <location>` — manage locations.
```
Comment (AmreenC) [inline README.md]: change 'set' to 'add'

File: README.md (hunk 3 above)
Comment (Lauraducky) [reply README.md]: I mean it sort of is and isn't an admin command

**Pull request merged**